File: fx-api-net/Data/Model/SpotRate.cs
namespace FundhouseFx.Data.Model
{
    public class SpotRate
    {
        public int SpotRateId { get; set; }
        
        public int BaseCurrencyId { get; set; }
        
        public int QuoteCurrencyId { get; set; }
        
        public int Timestamp { get; set; }
        
        public decimal Value { get; set; }
        
        public virtual Currency BaseCurrency { get; set; }
        
        public virtual Currency QuoteCurrency { get; set; }
    }
}

File: fx-api-php/packages/FundhouseFx/Data/Model/SpotRate.php
<?php
    namespace FundhouseFx\Data\Model;

    class SpotRate {
        public $spot_rate_id;
        public $base_currency_id;
        public $quote_currency_id;
        public $timestamp;
        public $value;

        function __construct($id, $base_currency_id, $quote_currency_id, int $timestamp, float $value)
        {
            $this->spot_rate_id = $id;
            $this->base_currency_id = $base_currency_id;
            $this->quote_currency_id = $quote_currency_id;
            $this->timestamp = $timestamp;
            $this->value = $value;
        }
    }
?>